thoth/python/package_version.py

```diff
@@ -245,10 +246,12 @@ def to_pipfile_lock(self) -> dict:
 
         return {self.name: result}
 
+    @Lazy
     def to_tuple(self) -> tuple:
         """Return a tuple representing this Python package."""
         return self.name, self.locked_version, self.index.url
 
+    @Lazy
     def to_tuple_locked(self) -> tuple:
```

Comment:
To keep codebase consistent, what about renaming these two to property style names? e.g. `tuple` and `tuple_locked`? The implementation might live with the current `to_tuple` and `to_tuple_locked` methods until we find all the occurrences and fix them.

Reply:
I like that :+1:

Reply:
I used as_tuple and as_tuple_locked because tuple is a keyword